core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.spec.ts

```diff
@@ -95,7 +88,7 @@ describe('Audio Bar Component', () => {
       .and.returnValue(mockOnAutoplayAudioEventEmitter);
     let secondaryTranslaionsSpy = spyOn(
       audioTranslationManagerService, 'setSecondaryAudioTranslations')
-      .and.returnValue(null);
+      .and.returnValue();
```

Comment:
Should we instead use `.and.callThrough();` , diito below?

Reply:
Done